.github/workflows/continuous-integration.yml

```diff
@@ -1,6 +1,6 @@
 name: CI tests
 
-on: [push, pull_request, release]
+on: [push]
```

Comment:
What was the reasoning behind removing `pull_request`, `release` triggers?

Reply:
The `release` trigger was meant to be used to test each realease automatically. But since this will never be executed on automated releases, I don't think it makes much sense to keep it.
The `pull_request` was meant to be used to test each PR before merging into master, but since `push` already tests that on every push, this is not necessary either.